# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../normalmap/makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gimp-normalmap/Makefile_orig	2005-11-16 06:52:44.000000000 +0100
+++ gimp-normalmap/Makefile	2005-11-16 22:20:03.000000000 +0100
@@ -15,12 +15,13 @@
 
 OBJS=normalmap.o preview3d.o scale.o
 
-LIBS=`pkg-config --libs gtk+-2.0 gtkglext-1.0 gimp-2.0 gimpui-2.0`
+LIBS=`pkg-config --libs gtk+-2.0 gtkglext-1.0 gimp-2.0 gimpui-2.0 pangox`
 ifdef WIN32
 LIBS+=-lglew32
 else
-LIBS+=-L/usr/X11R6/lib -lGLEW
+LIBS+=-L/usr/X11/lib -lGLEW
 endif
+PPATH=`$(GIMPTOOL) --gimpplugindir`
 
 all: $(TARGET)
 
@@ -31,7 +32,7 @@
 	rm -f *.o $(TARGET)
 	
 install: all
-	$(GIMPTOOL) --install-bin $(TARGET)
+	install $(TARGET) $(PPATH)/plug-ins/
 		
 .c.o:
 	$(CC) -c $(CFLAGS) $<
